Linear_cell_complex/include/CGAL/draw_linear_cell_complex.h

```diff
@@ -25,6 +25,7 @@
 
 #include <CGAL/Qt/Basic_viewer_qt.h>
 #include <CGAL/Random.h>
+#include <iostream>
```

Comment:
This include, as well as the others, should be *before* the `#ifdef CGAL_USE_BASIC_VIEWER`, because `std::cerr` and `std::endl` as used also, in the `#else` case.

Reply:
Done in https://github.com/CGAL/cgal/pull/2835/commits/8575968b878b290aa0356825d54a9e76a4b6d1f5